batch/batch/worker/worker.py

```diff
@@ -3043,6 +3044,13 @@ async def _initialize_jvms(self):
         assert all(jvmpool.full() for jvmpool in self._jvmpools_by_cores.values())
         log.info(f'JVMs initialized {self._jvmpools_by_cores}')
 
+    @staticmethod
+    def _initialize_jvms_postmortem(task: asyncio.Task):
+        try:
+            _ = task.result()
+        except Exception as e:
+            log.exception(f'JVMs not all initialized due to {type(e).__name__}')
```

Comment:
Slightly awkwardly worded so that this message contains the same “JVMs” and “initialized” keywords as other related log messages.